codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,7 +195,14 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the flow controllers.
+            connection.local().flowController(new DefaultHttp2LocalFlowController(connection, writer,
+                    DEFAULT_WINDOW_UPDATE_RATIO, autoRefillConnectionWindow));
+            connection.remote().flowController(new DefaultHttp2RemoteFlowController(connection));
```

Comment:
@nmittler - What if the user has already set the local and/or remote flow controller? They may also have alternative options configured on their flow controllers that this builder does not have access to.